namespaces/init.go

```diff
@@ -95,7 +103,11 @@ func Init(container *libcontainer.Config, uncleanRootfs, consolePath string, pip
 
 	label.Init()
 
-	if err := mount.InitializeMountNamespace(rootfs,
+	if (cloneFlags & syscall.CLONE_NEWNS) == 0 {
```

Comment:
@LK4D4 @mrunalp I understand what you say and agree with you. I think here is another situation. @LK4D4, in your code mount.InitializeMountNamespace() is executed if cloneFlags = 0 and container.MountConfig == nil, but it must not be executed in this case. Sorry if I don't see something obvious.